VideoLocker/src/main/java/org/edx/mobile/view/NavigationFragment.java

```diff
@@ -112,17 +113,30 @@ private void loadProfileImage(@NonNull ProfileImage profileImage, @NonNull Image
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
-        View layout = inflater.inflate(R.layout.drawer_navigation, container, false);
+        final View layout = inflater.inflate(R.layout.drawer_navigation, container, false);
 
-        TextView name_tv = (TextView) layout.findViewById(R.id.name_tv);
-        TextView email_tv = (TextView) layout.findViewById(R.id.email_tv);
+        final TextView name_tv = (TextView) layout.findViewById(R.id.name_tv);
+        final TextView email_tv = (TextView) layout.findViewById(R.id.email_tv);
+        final FrameLayout nameLayout = (FrameLayout)layout.findViewById(R.id.name_layout);
```

Comment:
Injecting is a bit more convenient and concise, but I guess if it's only being referenced in `onCreateView()` then it's fine to manually inflate it too. As for the data binding library, yeah we should integrate it as some point. You can open a JIRA issue on that if you wish.